Library/Homebrew/extend/kernel.rb

```diff
@@ -425,7 +425,7 @@ def ensure_formula_installed!(formula_or_name, reason: "", latest: false,
   end
 
   # Ensure the given executable is exist otherwise install the brewed version
-  def ensure_executable!(name, formula_name = nil, reason: "")
+  def ensure_executable!(name, formula_name = nil, reason: "", latest: false)
```

Comment:
It seems like the `latest` argument isn't used, i.e., it's not passed to `ensure_formula_installed!` – unless this is yet another Ruby thing that I'm not aware of.

Reply:
Whoops, good eye -- I missed that.